src/corona_hakab_model/consts.py

```diff
@@ -76,6 +76,24 @@ class Consts(NamedTuple):
     # [10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35],
     # [0.013,0.016,0.025,0.035,0.045,0.053,0.061,0.065,0.069,0.069,0.065,0.063,0.058,0.053,0.056,0.041,0.040,0.033,
     # 0.030,0.025,0.020,0.015,0.015,0.015,0.010,0.010]))
+    # state machine transfer probabilities
+    # probability of '...' equals (1 - all other transfers)
+    # it should always come last after all other transition probabilities were defined
+    latent_to_asymptomatic_prob = 0.3
+    latent_to_pre_symptomatic_prob = ...
+    pre_symptomatic_to_mild_condition_prob = ...
+    mild_to_close_medical_care_prob = 0.2375
+    mild_to_need_icu_prob = 0.0324
+    mild_to_pre_recovered_prob = ...
+    close_medical_care_to_icu_prob = 0.26
+    close_medical_care_to_mild_prob = ...
+    need_icu_to_deceased_prob = 0.0227
+    need_icu_to_improving_prob = ...
+    improving_to_need_icu_prob = 0.22
+    improving_to_pre_recovered_prob = 0.39
+    improving_to_mild_condition_prob = ...
+    pre_recovered_to_recovered_prob = ...
+    asymptomatic_to_recovered_prob = ...
```

Comment:
you need to add type hint(i think it's called like that) in order for it to work
example
improving_to_pre_recovered_prob: int = 0.39

Reply:
what's the type of '...'? also float?

Reply:
i think it's type(...)

Reply:
isn't that can cause problems when trying to give a float value to that parameter? e.g. if the user is trying to give all the probabilities in float and not using that option.

Reply:
you're right, it should be Union[float, type(...)],
my bad

Reply:
ok, cool